data/codes.json

```diff
@@ -175,7 +175,11 @@
         "PCS005": {
             "code": "PCS005",
             "message": "PCS005 : Verification mail sent, please check your mail"
-        }        
+        }      ,
```

Comment:
spacing, please keep it uniform with the rest of the contents in this file

Reply:
Done.